includes/admin/class-wcmypa-admin.php

```diff
@@ -989,14 +988,22 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
             ];
         }
 
-        return [
+        $confirmationData = [
             __("Delivery type:", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
-            __("Date:", 'woocommerce')                   => wc_format_datetime(new WC_DateTime($deliveryOptions->getDate())),
-            __("Extra options:", "woocommerce-myparcel") =>
-                sprintf("%s<br>%s",
-                        $deliveryOptions->getShipmentOptions()->hasSignature() ? $signatureTitle : null,
-                        $deliveryOptions->getShipmentOptions()->hasOnlyRecipient() ? $onlyRecipientTitle : null)
         ];
+
+        if (WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_SHOW_DELIVERY_DAY)) {
+            $array[__("Date:", 'woocommerce')] = wc_format_datetime(new WC_DateTime($deliveryOptions->getDate()));;
+        }
+
+        if ($deliveryOptions->getShipmentOptions()->hasSignature() || $deliveryOptions->getShipmentOptions()->hasOnlyRecipient()) {
+            $array[__("Extra options:", "woocommerce-myparcel")] =
```

Comment:
"Extra options:" zal vertaald worden. Tegenwoordig maken we gebruik van translation keys. Kijk maar eens bij settings_checkout_surcharge. Als je dit heb aangepast, dan zal je ook het drive document moeten aanpassen.

(dit ook graag bij Delivery type toepassen)

Reply:
beide staan er al op